build/ssvm.sh

```diff
@@ -8,5 +8,5 @@ if [[ "$OSTYPE" != "linux"* ]]; then
 fi
 
 mkdir -p build/_workspace/SSVM/build/tools/ssvm-evmc/
-wget -O build/_workspace/SSVM/build/tools/ssvm-evmc/libssvmEVMC.so \
+[[ -f build/_workspace/SSVM/build/tools/ssvm-evmc/libssvmEVMC.so ]] || wget -O build/_workspace/SSVM/build/tools/ssvm-evmc/libssvmEVMC.so \
```

Comment:
```suggestion
[[ -f build/_workspace/SSVM/build/tools/ssvm-evmc/libssvmEVMC.so ]] && exit 0
wget -O build/_workspace/SSVM/build/tools/ssvm-evmc/libssvmEVMC.so \
```

I don't really mind on this one and wouldn't have noticed if it wasn't like this on other changes of same commit. No need to change it, both work fine.